site/docs/mfa/callbacks/about.mdx

```diff
@@ -0,0 +1,45 @@
+  - mfa
+  - callbacks
+  - bandwidth
+hide_title: false
+sidebar_label: About
+custom_edit_url: github.com/Bandwidth/api-docs/tree/main/site/docs/mfa/callbacks/about.mdx
+---
+# Two-Factor Authentication Callbacks
+Two-Factor Authentication (2FA) Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) rely on DLRs (Delivery Receipts) sent to the Two-Factor Authentication (2FA) platform by both Bandwidth's North America and International platform. Since these two platforms send different DLR formats, the 2FA platform standardizes them into a singular format for simplicity and ease of consumption. The 2FA platform then takes these standardized DLRs and sends them via an HTTP POST to the specified URL that was set in the `callbackUrl` field in the [initial request](../../messaging/quickStart.mdx#configuring-callback-urls). The `callbackUrl` field can be any publicly addressable url.
+
+## Message Callbacks Concepts
+- Both North America and International callbacks have the same fields and have the same format. However, some of these fields differ in values to denote between the two systems.
+- North America callbacks are retried for 24 hours if the 2FA platform is unable to deliver it to you.
+- International callbacks are not retried if the 2FA platform is unable to deliver it to you after the initial delivery attempt.
+
+## North America Callbacks
+- The 2FA platform considers any message that is sent to a phone number with the country code `+1` to be North America.
+- The type of North America callbacks are as follows:
+  - [message-delivered](./northAmerica/messageDelivered.mdx) - Message has been delivered to the carrier.
+  - [message-failed](./northAmerica/messageFailed.mdx) - Message failed to deliver.
+
+## International Callbacks
+- The 2FA platform considers any message that is sent to a phone number that does not have the country code `+1` to be international.
+- The type of international callbacks are as follows:
+  - [message-delivered](./international/messageDelivered.mdx) - Message has been delivered to the carrier.
+  - [message-failed](./international/messageFailed.mdx) - Message failed to deliver.
+
+## Two-Factor Authentication Provisioning
+When Two-Factor Authentication is enabled on your account, the 2FA platform generates a separate Application in the Bandwidth Dashboard specifically for 2FA. As part of this application, there is a field present called `Callback URL` that is auto-populated with the default Callback URL for 2FA. Having this default endpoint ensures proper logging of requests to the platform for better serviceability, as well as simplifies the sending of different types of DLRs from messaging. The 2FA platform proxies all callbacks to the destination of your choice using the value in the `callbackUrl` field.
+
+See the below image as an example of the Callback URL from the Bandwidth Dashboard.<br/>
+<br/>
+<img src="../../../static/img/mfa-callbackUrlExample.png"></img>
```

Comment:
This image link is broken in the autogen sample deployment http://bw-api-docs-edge-1228.s3-website-us-east-1.amazonaws.com/docs/mfa/callbacks

Reply:
updated